src/gui/executionlogwidget.h

```diff
@@ -30,33 +30,29 @@
 #define EXECUTIONLOGWIDGET_H
 
 #include <QWidget>
+
 #include "base/logger.h"
 
-class LogListWidget;
 
 namespace Ui
 {
     class ExecutionLogWidget;
 }
 
+class LogFilterModel;
+
 class ExecutionLogWidget : public QWidget
 {
     Q_OBJECT
 
 public:
-    ExecutionLogWidget(QWidget *parent, const Log::MsgTypes &types);
-    void showMsgTypes(const Log::MsgTypes &types);
+    ExecutionLogWidget(QWidget *parent, Log::MsgTypes types);
+    void showMsgTypes(Log::MsgTypes types);
     ~ExecutionLogWidget();
```

Comment:
constructor & destructor should be placed together.

Reply:
done.